doc/03_reference/pipeline_details.rst

```diff
@@ -26,7 +26,7 @@ See Multi- and Single-Cycle Instructions below for the details.
 Third Pipeline Stage
 --------------------
 Ibex can be configured to have a third pipeline stage (Writeback) which has major effects on performance and instruction behaviour.
-This feature is *EXPERIMENTAL* and the details of its impact are not yet documented here.
+The details of its impact are not yet documented here.
```

Comment:
Should we track this in a separate issue?

Reply:
There are various doc updates still to do, including this, I'll create an issue to track everything I can think of.